skate-plugin/src/test/kotlin/com/slack/sgp/intellij/ChangeLogParserTest.kt

```diff
@@ -21,9 +21,9 @@ import org.junit.Test
 
 class ChangeLogParserTest {
```

Comment:
An issue with the previous tests is that they tested the format of the `CHANGELOG.md` file in this repo, which used this format

```
0.1.0
-----

_2023-05-05_

- Some update
```

when the actual format we parse in `WHATSNEW.md` files is

```

```
2023-05-05
----------

- Some update
```
```

Reply:
Hey @ZacSweers , I do see Italics `_` in WHATSNEW.md in the Android repo. Should that be changed? There is only one entry at the moment:
------------------------
What's New
==========

_2023-08-08_
------------

### Android Studio ....
------------------------

Reply:
Yep we'll need to fix those, they were added because of this confusion I think heh

Reply:
😅 kk, I'll approve 👍